README.md

```diff
@@ -4,7 +4,7 @@ Bundle for interfacing UFS models with JEDI interfaces
 
 ## Required thirdparty libraries
 
-This bundle requires the following spack-stack modules be loaded (all except `fms@2022.04` are in the `skylab-3.0.0` environment):
+This bundle requires the following spack-stack modules be loaded (all except `fms@2022.04` and `atlast@0.35.1` are in the `skylab-3.0.0` environment):
```

Comment:
We need to update this section (and the list of modules) to something more recent